packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IconProps } from '@fluentui-react-native/icon';
+import { IButtonProps, IButtonTokens } from '@fluentui-react-native/button';
+
+export const tabsItemName = 'TabsItem';
```

Comment:
Did we do research on what this should be called? IMO "tab header" sounds more intuitive, but I could go either way

Reply:
We named it Tabs**Item** taking from XAML and Fluent UI using Pivot**Item**. Also, Fluent UI seems to be moving to [TabItem](https://github.com/microsoft/fluentui/blob/master/packages/react-tabs/src/components/Tabs/TabItem.tsx).